docker/centos-graal-maven-s2i/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# [Quarkus.io](http://quarkus.io) *native* S2I builder
+# Original Author: Michael Vorburger.ch
+
+# This part is heavily inspired by ../centos-graal/Dockerfile
+FROM centos:latest
```

Comment:
Do we really want to use "latest"?

Reply:
Using e.g. `centos:7` is actually meaningless and doesn't work the way you think it does, as far as I understand... I learnt this the hard way over here https://github.com/fabric8io-images/s2i/issues/206#issuecomment-445047444 and here https://github.com/fabric8io-images/s2i/issues/172 full background logged here https://gist.github.com/vorburger/ea367a76f69aafb8260c62f6bedf2788... Unless all that's only for 7.5/7.6 and not 8, but I doubt it. I am still happy to change it to `centos:7` anyway if you like, but doubt that's really useful and does what most expect.